src/lib/discord/commando/commands/tracked.js

```diff
@@ -140,7 +140,7 @@ exports.run = async (client, msg, command) => {
 
 		try {
 			const hastelink = await client.hastebin(message)
-			return await msg.reply(`${target.name} ${client.translator.translate('tracking list is quite long. Have a look at')} ${hastelink}`)
+			return await msg.reply(`${client.translator.translate('tracking list is quite long. Have a look at')} ${hastelink}`)
```

Comment:
Why did you remove the users name here?

Reply:
Basically when using `!tracked` it's you that's asking for tracked list so why to list your name there? It just doesn't look right

Reply:
I liked having the name there, made it look like a conversation - I wouldn't remove it. If removed then `Tracking list...` needs upper case for T

Reply:
It doesn't look like conversation at all since there are already lines about notifications enabled, areas and location before thet part. I'll change that `t` then. Anyway I don't mind if you undo my changes at that part